DNN Platform/Website/admin/Skins/Logo.ascx.cs

```diff
@@ -58,6 +64,40 @@ protected override void OnLoad(EventArgs e)
                         {
                             this.imgLogo.ImageUrl = imageUrl;
                             logoVisible = true;
+                        }
+
+                        if (this.InjectSVG.GetValueOrDefault() == true)
+                        {
+                            logoVisible = false; // hide the <img>, we are going to inject an <svg> tag instead.
+
+                            hypLogo.CssClass = this.CssClass; // put the custom CSS class on the hyperlink instead

```

Comment:
This provides an inconsistency for the designer by assigning the CSS class either to the image itself or the hyperlink if SVG.  Is this a desired/acceptable behavior change  @david-poindexter I'd love your $0.02 on this change

Reply:
This was on purpose.  It thought it desirable to include the CssClass somewhere and there was nowhere else to put it unless we want to create another html tag.  

Also, putting it on the <a> tag is at least consistent with how BootStrap expects the markup to look.
  <a class="navbar-brand" href="#">Navbar</a>

Since this is new functionality I think we have greenfield to do what we want here...